rspec-mode.el

```diff
@@ -635,6 +651,15 @@ file if it exists, or sensible defaults otherwise."
          (substring file (length remote))
        file))))
 
+(defun rspec--vagrant-wrapper (command)
+  (if (rspec-vagrant-p)
+      (format "vagrant ssh -c 'cd %s; %s'"
+              rspec-vagrant-cwd
+              (replace-regexp-in-string (rspec-project-root)
+                                        rspec-vagrant-cwd
```

Comment:
I made the change, but I'm not sure on how to make `rspec-vagrant-p` be called only once. Did you mean to pass it as an argument to `rspec--shell-quote-local`? The issue I see is that the function is called in several places and it's hard for me to tell where exactly I should make the call. But I might be misunderstanding you.